### PR TITLE
Add a setup for a local deployment

### PR DESCRIPTION
**Pull request by Xarthisius**
I just wanted to put it up here, for comments and suggestions.

### CODE REVIEW

**Comment by matthewturk** [comment]
This is awesome and I would love to see this included!

At some point do we need to rethink the name of this repo?

**Comment by Xarthisius** [comment]
@MatthewTurk always treat names that I come up with as placeholders :) I'm happy to change if you have any suggestions

File: wt_dev/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+Setting up local testing instance of Girder
+===========================================
+
+Requirements:
+
+ * docker >= 18.04
```
Comment (craig-willis) [inline wt_dev/README.md]: Worked for me with 17.10.0-ce on CoreOS

File: wt_dev/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ * GitHub app OAuth credentials with callback set to `http://girder.vcap.me/api/v1/oauth/github/callback`.
```
Comment (craig-willis) [inline wt_dev/README.md]: Add link to Github docs? https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/

**Review by craig-willis — APPROVED** [review]
Overall, looks great.  If merged, I can spend some time on improving the README.

**Pull request merged**